python/ray/tune/search/skopt/skopt_search.py

```diff
@@ -75,6 +75,8 @@ class SkOptSearch(Searcher):
         convert_to_python: SkOpt outputs numpy primitives (e.g.
             ``np.int64``) instead of Python types. If this setting is set
             to ``True``, the values will be converted to Python primitives.
+        optimizer_kwargs: Parameters to pass to the SkOpt optimizer.
```

Comment:
Can you add/update a unit test in `test_searchers` for the new functionality? Thanks!

Reply:
Thanks for your suggestion and updated a unit test there.

Reply:
@zhiruiluo actually would it make sense to create a different test? Want to make sure that (1) the original case doesn't regress, and (2) that the args are actually being respected.